Sources/NautilusTelemetry/Metrics/HistogramValues.swift

```diff
@@ -26,6 +26,11 @@ struct HistogramBuckets<T: MetricNumeric> {
 	var data: [UInt64]
 	let explicitBounds: [T]
 
+	var isEmpty: Bool {
+		guard !data.isEmpty else { return true }
+		return data.allSatisfy { $0 == 0 }
```

Comment:
This might be more clear:

```suggestion
		return data.isEmpty || data.allSatisfy { $0 == 0 }
```

Reply:
Sure.